test/functional/page_objects/visualize_page.js

```diff
@@ -681,10 +680,10 @@ export function VisualizePageProvider({ getService, getPageObjects, updateBaseli
       const lastRow = await table.findByCssSelector('tr:last-child');
       const fromCell = await lastRow.findByCssSelector('td:first-child input');
       await fromCell.clearValue();
-      await fromCell.type(`${from}`);
+      await fromCell.type(`${from}`, { charByChar: true });
       const toCell = await lastRow.findByCssSelector('td:nth-child(2) input');
       await toCell.clearValue();
-      await toCell.type(`${to}`);
+      await toCell.type(`${to}`, { charByChar: true });
```

Comment:
Firefox has typing issue in these inputs, typing char by char solves it.

Reply:
Starting to think `charByChar` should be the default, especially since it more closely resembles a user.

Reply:
I was thinking about it as well, at least to have by default on Firefox. Can be a sensitive change, so let's have it tested in separate PR